### ISSUE TITLE
Pip not installing dependencies

### ISSUE THREAD

**Issue by robertskmiles** [deepfates/memery#11]
First time opening an issue like this, please let me know if I do something wrong!

Pip seems to be not installing the dependencies for this package:

```
$ pip install memery
Defaulting to user installation because normal site-packages is not writeable
Collecting memery
  Using cached memery-0.0.2-py3-none-any.whl (14 kB)
Installing collected packages: memery
Successfully installed memery-0.0.2
```
The `pip install` command completes very quickly.

```
rob@tortuga ~/Pictures$ memery . 'music' -n 2
Traceback (most recent call last):
  File "/home/rob/.local/bin/memery", line 5, in <module>
    from memery.cli import __main__
  File "/home/rob/.local/lib/python3.9/site-packages/memery/cli.py", line 6, in <module>
    import typer
ModuleNotFoundError: No module named 'typer'
```

I then installed `python-typer` (from pacman), and tried again:

```
rob@tortuga ~/Pictures$ memery . 'music' -n 2
Traceback (most recent call last):
  File "/home/rob/.local/bin/memery", line 5, in <module>
    from memery.cli import __main__
  File "/home/rob/.local/lib/python3.9/site-packages/memery/cli.py", line 7, in <module>
    from .core import queryFlow
  File "/home/rob/.local/lib/python3.9/site-packages/memery/core.py", line 8, in <module>
    from .loader import get_image_files, archive_loader, db_loader, treemap_loader
  File "/home/rob/.local/lib/python3.9/site-packages/memery/loader.py", line 18, in <module>
    import torchvision
ModuleNotFoundError: No module named 'torchvision'
```

So I installed that, and now it doesn't have `annoy`. At this point I opened the issue.

Some other info:
```
rob@tortuga ~/Pictures$ uname -a
Linux tortuga 5.4.116-1-MANJARO #1 SMP PREEMPT Sun May 2 11:10:35 UTC 2021 x86_64 GNU/Linux
rob@tortuga ~/Pictures$ pip --version
pip 20.3.1 from /usr/lib/python3.9/site-packages/pip (python 3.9)
rob@tortuga ~/Pictures$ python --version
Python 3.9.4
```

Thanks, and let me know if there's any other info that would be helpful

**Comment by deepfates** [comment]
Thank you! You did great. This was definitely a big error on my part and you finding it helped a lot :sweat_smile: 

The package downloaded was just my tiny amount of code without dependencies. This was due to the `nbdev` build practices which use `settings.ini`, instead of `requirements.txt`. I found the right way to package things for the nbdev+pip build system, so if you start over at `pip install memery` it should grab all the dependencies this time.

NB I'm not sure how memery will interact with your python 3.9 environment. i've been testing with 3.7 but i don't remember why
